### ISSUE TITLE
Add GAppLaunchContext support

### ISSUE THREAD

**Issue by rossburton** [clutter-project/mx#19]
GTK+ has GdkAppLaunchContext so that starting an app will use startup notification and so on.  MX (or Clutter) should do the same, probably by copying most of GdkAppLaunchContext.

**Comment by thos** [comment]
Startup notification support is already provided by an implementation in MxApplication. Is there something missing from that implementation?

**Comment by rossburton** [comment]
Yes, I want to start another application from my Mx application and have startup notification for it work correctly.  This means using GAppInfo to find the application to launch and passing a GAppLaunchContext implementation to integrate with the system.
